scripts/particle_rugosity/ReadMe.md

```diff
@@ -0,0 +1,85 @@
+# Particle Rugosity
+
+## Summary
+Simulates the particle shape of the given crystal structure using BFDH mophology 
+prediction, then calculates the particle rugosity by determining the rugosity 
+of the facets and calculating the weighted average (by facet area).
```

Comment:
Could you please mention in here that it uses the lowest rugosity for a given hkl to make sure people are aware that the offset is iterated through?